static/sass/_snapcraft_p-featured-snap.scss

```diff
@@ -0,0 +1,88 @@
+@mixin snapcraft-p-featured-snaps {
+  $light-transparent: rgba(255, 255, 255, 0);
+  $light-semi-opaque: rgba(255, 255, 255, .65);
+  $light-opaque: rgba(255, 255, 255, .8);
+  $dark-transparent: rgba(0, 0, 0, 0);
+  $dark-semi-opaque: rgba(0, 0, 0, .65);
+  $dark-opaque: rgba(0, 0, 0, .8);
+
+  .p-generated-featured-snap {
```

Comment:
The naming is a bit confusing... We have `p-media-object--is-featured`, `p-featured-snap`, `p-media-object--is-featured` `snapcraft-p-grid--featured`... And I don't really know which of these are part of same pattern, which are different, are they meant to be used together or separately?

Reply:
:+1: I've reduced this down and used the wrapping `snapcraft-p-grid--featured` to alter sizes

Reply:
Thanks. It's better now.
But I still don't like the `p-generated-featured-snap` name. Why "generated"? I would guess it's because we "generate" a colour background for it, but I don't think this implementation detail should define the name of the pattern.

I know we have 'featured' snap elsewhere, so this would need to be something different... Maybe it could have something to do with banner? p-snap-banner? or p-featured-banner? As I suppose in future we could or will show snap banner images as background if provided?